config/environments/production.rb

```diff
@@ -72,12 +72,18 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
+
+
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
+  config.active_record.yaml_column_permitted_classes = [Symbol]
+
```

Comment:
Why do we only need to add Symbol here but a lot of classes in an earlier location?

Reply:
I don't think it is needed will test if removing that works